api/HTMLScriptElement.json

```diff
@@ -330,6 +330,53 @@
           }
         }
       },
+      "integrity": {
+        "__compat": {
+          "support": {
+            "chrome": {
+              "version_added": "45"
+            },
+            "chrome_android": {
+              "version_added": "45"
+            },
+            "edge": {
+              "version_added": "17"
+            },
+            "firefox": {
+              "version_added": "43"
+            },
+            "firefox_android": {
+              "version_added": "43"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "32"
+            },
+            "opera_android": {
+              "version_added": "32"
+            },
+            "safari": {
+              "version_added": "11.1"
+            },
+            "safari_ios": {
+              "version_added": "11.3"
```

Comment:
The Safari iOS data for the content attribute is false. I'm guessing that's just wrong, can you update that and other bits of html/elements/script.json to match?

Because it was all added in https://trac.webkit.org/changeset/216347/webkit, we can be confident the ranges should be the same.